ros_gz_sim/launch/gz_spawn_model.launch

```diff
@@ -0,0 +1,28 @@
+<launch>
+  <arg name="world" default="" />
+  <arg name="file" default="" />
+  <arg name="xml_string" default="" />
+  <arg name="topic" default="" />
+  <arg name="name" default="" />
+  <arg name="allow_renaming" default="" />
+  <arg name="x" default="" />
+  <arg name="y" default="" />
+  <arg name="z" default="" />
+  <arg name="roll" default="" />
+  <arg name="pitch" default="" />
+  <arg name="yaw" default="" />
+  <gz_spawn_model 
+    world="$(var world)"
+    file="$(var file)"
+    xml_string="$(var xml_string)"
+    topic="$(var topic)"
+    name="$(var name)"
+    allow_renaming="$(var allow_renaming)"
+    x="$(var x)"
+    y="$(var y)"
+    z="$(var z)"
+    roll="$(var roll)"
+    pitch="$(var pitch)"
+    yaw="$(var yaw)"
```

Comment:
This looks wrong, isn't it?
```suggestion
    yaw="$(var yaw)">
```

Reply:
Typo fixed in [04c9dc8](https://github.com/gazebosim/ros_gz/pull/551/commits/04c9dc8585abac796b68987a5f25c78677c656a4).